### PR TITLE
Added Status badges

### PR DESCRIPTION
**Pull request by KarishmaGhiya**
## Description

Added Status badges for client and management APIs
---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

@Azure/azure-sdk-eng

### CODE REVIEW

**Review by scbedd — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Microsoft Azure SDK for .NET
- ----------
+
+[![Build Status](https://travis-ci.org/Azure/azure-sdk-for-net.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-net) [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/net/azure-sdk-for-net%20-%20client?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=41&branchName=master)
```
Comment (weshaggard) [inline README.md]: Can you split this one up and label them as management and client? Similar to the java repo https://github.com/Azure/azure-sdk-for-java/pull/2605/files.

Comment (weshaggard) [reply README.md]: Also while looking at the Java readme I noticed that the name of the build definition is part of the URL which we just changed so it is pointing at some old unknown state. According to the devops docs (https://docs.microsoft.com/en-us/rest/api/azure/devops/build/status/get?view=azure-devops-rest-5.0) and a quick test it appears that we can instead use definition id so can you please update this to use the id instead, as well as the other repos so we can prevent ourselves from breaking these if we change the name.

Comment (weshaggard) [reply README.md]: For consistency between the status badge and the link we should also include the branch name. So for this one I'd have `https://dev.azure.com/azure-sdk/internal/_apis/build/status/41?branchName=master`

Comment (weshaggard) [reply README.md]: I just noticed this one did have the branch already but the java one didn't. We should try to make them all 4 consistent and have them use the definition id and branch name.

**Review by scbedd — APPROVED** [review]
Still displays, and both links work for me.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**